app/concepts/cinema_halls/use_cases/create.rb

```diff
@@ -0,0 +1,15 @@
+module CinemaHalls
+  module UseCases
+    class Create
+      attr_reader :repository
+
+      def initialize(repository: CinemaHalls::Repository.new)
+        @repository = repository
```

Comment:
Is this a place for Processor? I'm not sure I get the idea of this element of concepts.